first-service/src/main/resources/db.migration/V1_0_0__car_init.sql

```diff
@@ -0,0 +1,16 @@
+create table cars(
+    id           serial primary key,
+    brand        VARCHAR(63),
+    body_type    VARCHAR(63),
+    wheel_radius DOUBLE PRECISION,
+    deleted      boolean   default false,
+    created_at   TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at   TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+INSERT INTO cars (brand, body_type, wheel_radius)
```

Comment:
it is not a good practice to have migration and insertion in one place

Reply:
Splitted for now in PR "Fixed issues from Github"